basic-examples/quoting/README.md

```diff
@@ -1,15 +1,16 @@
-# Expressions
+# Quoting
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates example expressions with quotes as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+There are multiple types of quoting, and quotes must pass through YAML's syntax. Look at this example to learn about the various ways to determine what would work best for your use case.
 

```

Comment:
If you're going to mention YAML's processing, we should probably point out that ARCA processes quoting, too, which is why half of the example strings appear to be quoted _twice_ (the other half are quoted only once because the block flow provides implicit quoting for YAML).  And, you might want to point out that this makes it challenging to embed quotation marks into the string.

And, as I said, (if you agree) it might be worth putting in a plug here for using block flow and raw strings when embedding quotation marks.

Reply:
I made some changes. Let me know if it's better.